Dockerfile

```diff
@@ -20,46 +20,25 @@ RUN mkdir -p $ADAPTER_ROOT/
 
 COPY . $ADAPTER_ROOT/
 
+WORKDIR $ADAPTER_ROOT
 # If LOCAL_AETHER_MODELS was used, then patch the go.mod file to load
 # the models from the local source.
 RUN if [ -n "$LOCAL_AETHER_MODELS" ] ; then \
     echo "replace github.com/onosproject/aether-models/models/aether-4.x => ./local-aether-models/aether-4.x" >> $ADAPTER_ROOT/go.mod; \
     echo "replace github.com/onosproject/aether-models/models/aether-2.0.x => ./local-aether-models/aether-2.0.x" >> $ADAPTER_ROOT/go.mod; \
+    echo "replace github.com/onosproject/aether-models/models/aether-2.1.x/v2 v2.1.13 => ./local-aether-models/aether-2.1.x" >> $ADAPTER_ROOT/go.mod; \
+    go mod tidy;\
```

Comment:
I think we should remove this line because doing this inside Dockerfile bypasses the need to doing it in the source code. The idea is to have what Docker uses inline with what the source code has. For example, as you can see in [this PR](https://github.com/omec-project/sctplb/pull/11), `go mod tidy` is executed inside the Dockerfile which wrongly hides the need to execute `go mod tidy` in the source code

```suggestion
```